albumentations/core/composition.py

```diff
@@ -269,6 +270,65 @@ def __call__(self, force_apply=False, **data):
         return data
 
 
+class SomeOf(BaseCompose):
+    """Select some of the transforms to apply. Selected transforms will be called with `force_apply=True`.
+    Transforms probabilities will be normalized to one 1, so in this case transforms probabilities works as weights.
+
+    Args:
+        transforms (list): list of transformations to compose.
+        sample_range (int, int): Range of possible sample sizes.
+        p (float): probability of applying selected transform. Default: 0.5.
+    """
+
+    def __init__(self, sample_range: Tuple[int], transforms, p=0.5):
+        super().__init__(transforms, p)
+        if isinstance(sample_range, (list, tuple)):
```

Comment:
Wrong `sample_range` type. Below you check that `sample_range` may be list. Better to use type `Sequence`.

Reply:
And better to check is `sample_range` is instance of `Sequence`